test/tools/tool2/tool2.xml

```diff
@@ -27,6 +27,9 @@
 
 **Input**
 
+This will make the linter test fail.
```

Comment:
Hmm, it doesn't seem to report this

Reply:
OK. The linter actually complains about `ERROR: Error 'HTTPConnectionPool(host='bugus.url', port=80): Max retries exceeded with url: /for_test (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb1a8342d50>: Failed to establish a new connection: [Errno -2] Name or service not known'))' accessing http://bugus.url/for_test` .. 

So we need to use an existing host but invalid path for this .. or just check for this. I would go for the latter because we just want to know if the linter can fail.